.travis.yml

```diff
@@ -6,14 +6,16 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - hhvm
+matrix:
+  include:
+    - php: 5.3
+      dist: precise
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 7.0
+    - php: 7.1
```

Comment:
what about 7.2 ?

Reply:
Good idea, I'll add it in another PR :) .